protocol.md

```diff
@@ -1213,17 +1213,29 @@ _Response_:
 * result: [`Location`](#location) | [`Location`](#location)[]
 * error: code and message set in case an exception happens during the definition request.
 
-#### Get Symbol Request
+#### Resolve Symbol Request
 
-The get symbol request is sent from the client to the server to resolve the definition location and symbol information for a given text document position.
+The resolve symbol request is sent from the client to the server to resolve the definition location, symbol information, and related symbols for a given text document position.
 
 _Request_
-* method: 'textDocument/symbol'
+* method: 'textDocument/resolveSymbol'
 * params: [`TextDocumentPositionParams`](#textdocumentpositionparams)
 
 _Response_:
-* result: `SymbolInformation` | `SymbolInformation`[]
-* error: code and message set in case an exception happens during the definition request.
+* result: `ResolvedSymbolInformation` | `ResolvedSymbolInformation`[], defined as follows:
+```typescript
+/**
+ * Represents a symbol node in an abstract syntax tree,
+ * with pointers to related symbols.
+ */
```

Comment:
Maybe remove the bit about AST, since it seems this information could be derived from anything, including (but not limited to) an AST.